frontend/common/src/components/MissingSkills/MissingSkills.tsx

```diff
@@ -1,11 +1,61 @@
 import React from "react";
 import { useIntl } from "react-intl";
 
+import {
+  ExclamationTriangleIcon,
+  LightBulbIcon,
+} from "@heroicons/react/24/solid";
 import Chip, { Chips } from "../Chip";
 
 import { getMissingSkills } from "../../helpers/skillUtils";
 import { getLocale } from "../../helpers/localize";
 import type { Maybe, Skill } from "../../api/generated";
+import { PillColor, PillMode } from "../Pill";
+
+const MissingSkillsBlock = ({
+  pillType,
+  title,
+  blurb,
+  icon,
+  missingSkills,
+  ...rest
+}: {
+  pillType: { color: PillColor; mode: PillMode };
+  title: string;
+  blurb: string;
```

Comment:
These should be `React.ReactNode` to accept the results of `intl.formatMessage`.

```suggestion
  title: React.ReactNode;
  blurb: React.ReactNode;
```